app/code/Magento/Review/Model/ResourceModel/Rating.php

```diff
@@ -48,12 +54,14 @@ public function __construct(
         \Magento\Framework\Module\Manager $moduleManager,
         \Magento\Store\Model\StoreManagerInterface $storeManager,
         \Magento\Review\Model\ResourceModel\Review\Summary $reviewSummary,
+        \Magento\Framework\App\State $state,
```

Comment:
Due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add new required parameters to constructor.

Reply:
Hello @VladimirZaets , I'll check and update it asap.